src/components/QuranReader/index.tsx

```diff
@@ -38,18 +40,19 @@ const verseFetcher = async (input: RequestInfo, init?: RequestInit) => {
 
 const QuranReader = ({ initialData, chapter }: QuranReaderProps) => {
```

Comment:
while changing translations, there's a brief moment where the default translation is shown, regardless of which translations are currently selected.

Is there a way to either: 
1) show the current translations until the new ones load
2) show a loading indicator instead of the default translations. This loading indicator can just be `loading...` text for now

Reply:
In a separate PR, do the following changes:

1. In the redux store, rename translations to selectedTranslations
2. Add a state called isUsingDefaultTranslations, this checks whether the currently selected translations match the default ones, if yes, it'll be set to true, otherwise will be set to false.
3. If isUsingDefaultTranslations is false, do not use initial data
4. If there's no data (and no initial data) show a loading indicator